build-tools/build.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+script_start_time=$SECONDS
+# ------------
+# ECHO LICENSE
+# ------------
+echo "::group::License"
+cat /LICENSE
+echo "::endgroup::"
+
+# ----------------
+# SETTING VARIABLES
+# ----------------
+
+set -e # Exit on error
+
+git config --global --add safe.directory /github/workspace
+
+
+# ----------------
+# CHECK IF TEMPLATE
+# ----------------
+echo "::group::Checking if this is a template"
+
+template=$(awk -F'=' '/^IS_LIBRARY:=/{print $2}' Makefile)
+if [ "$template" == "1" ]; then
+    echo "is template"
+else
+    echo "is not template"
+fi
+echo "template=$template" >> $GITHUB_OUTPUT
```

Comment:
Isn't this for use in other gh workflow steps, so its unnecessary here as there is only one step?

Reply:
yeah, it was there from the old action. I removed it.